docs/CONFIG_FILES.md

```diff
@@ -5,8 +5,8 @@ configuration. Data mapping configuration can be reused across multiple indexing
 
 - All config files live in the [main resources directory](../underlay/src/main/resources/config/) of the `underlay` Gradle sub-project.
 - The config schemas are defined in Java classes in the [serialization package](../underlay/src/main/java/bio/terra/tanagra/underlay/serialization/).
-- Documentation for all available properties in the config schemas are generated from the inline annotations as 
-- [markdown](generated/UNDERLAY_CONFIG.md).
```

Comment:
Can you also add links to the new generated markdown file to the top-level README.md?

Reply:
Done